codespell_lib/data/dictionary.txt

```diff
@@ -8652,6 +8679,7 @@ iwth->with
 januar->january
 janurary->january
 Januray->January
+japaneese->japanese
```

Comment:
The correction should be capitalised.

Reply:
both done

Reply:
Thanks, I'm not sure what the second one was.